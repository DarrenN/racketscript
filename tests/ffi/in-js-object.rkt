#lang racketscript/base

(require racketscript/interop)

(define obj {$/obj [name        "John"]
                   [city        "San Jose"]
                   [occupation  "Driver"]})

(for ([(k v) (in-js-object obj)])
  (displayln (list k v)))

(displayln 
  (for/hash ([(k v) (in-js-object obj)])
            (values k v)))
