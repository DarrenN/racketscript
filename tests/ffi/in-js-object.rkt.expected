(name John)
(city San Jose)
(occupation Driver)
#hash((occupation . Driver) (city . San Jose) (name . John))
